### ISSUE TITLE
How it is works ?

### ISSUE THREAD

**Issue by Someonetoldme584** [foxide123/AndroidWearOSChatGPT#1]
How it is works ?

**Comment by foxide123** [comment]
Hello, let me first check the repository if everything is working correctly and then I will update the README explaining how it works

**Comment by Someonetoldme584** [comment]
I downloaded to my watch and i clicked authencite nothing happened

**Comment by foxide123** [comment]
![20230916_222522](https://github.com/foxide123/AndroidWearOSChatGPT/assets/66741324/54ee41a3-5bf5-4fec-a843-590c5484a9d0)
It should display code similar to the one from the image. The app was configured for galaxy watch so that the confirmation website is opened on the phone where you have to paste the code

**Comment by Someonetoldme584** [comment]
What website i opened clicked the button and i saw the code but i couldnt see a site ?

Samsung galaxy watch 6 classic

**Comment by foxide123** [comment]
Do you have Galaxy Wearable application installed on your phone?

**Comment by Someonetoldme584** [comment]
Yes

**Comment by foxide123** [comment]
![Screenshot_20230916_230719_Samsung Internet](https://github.com/foxide123/AndroidWearOSChatGPT/assets/66741324/d0ed5678-e936-451e-b2d9-caa17e1b808f)

If the application is connected to the watch then it should open the website like the one from the image on your phone. It can only opened the website on the phone and not on the smartwatch. But you have to do it only once

**Comment by foxide123** [comment]
Also you will have to create new api key on this website https://platform.openai.com/account/api-keys and then replace it in app>build.gradle (lines 19 and 25)

**Comment by Someonetoldme584** [comment]
Nothing happened chat opened  im clicking send button nope ? 

btw site didnt opened i skipped the auth idk how

**Comment by Someonetoldme584** [comment]
Ok i did everything it works but there is one problem its hard to read the what chatgpt writes

**Comment by Someonetoldme584** [comment]
And the tts is reads only in english can we change to turkish ?

**Comment by foxide123** [comment]
Turkish is not supported. I will fix the screen in the future so it is more readable. If you have any other suggestions let me know. I will probably delete the record button since you have already simillar option in the touchpad.

**Comment by foxide123** [comment]
Also I have to add the option to choose the gpt model since right now it uses the most powerful (and thus most expensive one).

**Comment by Someonetoldme584** [comment]
yup thank youu and yo shoul add this  program to google play store
